h/templates/client/thread.html

```diff
@@ -45,3 +45,7 @@
       ng-show="vm.shouldShowAsReply()">
   </li>
 </ul>
+
+<footer class="thread-anchor-notice" ng-if="vm.container.message.$orphan">
```

Comment:
Yeah, I had found that but it seemed to be that way in your original commit, ok, that fixes it, thanks!